stdlib/public/Concurrency/TaskSleep.swift

```diff
@@ -287,4 +294,104 @@ extension Task where Success == Never, Failure == Never {
       throw error
     }
   }
+
+  @available(macOS 9999, iOS 9999, tvOS 9999, watchOS 9999, macCatalyst 9999, *)
+  static func sleep(until seconds: UInt64, _ nanoseconds: UInt64, clockId clock: swift_clock_id) async throws {
```

Comment:
Nits:
- Availability macro
- Leading underscore rule
- Explicit access modifier

```suggestion
  @available(SwiftStdlib 9999, *)
  internal static func _sleep(until seconds: UInt64, _ nanoseconds: UInt64, clockId clock: swift_clock_id) async throws {
```